sputnikdao-factory2/src/lib.rs

```diff
@@ -161,6 +203,7 @@ pub fn slice_to_hash(hash: &[u8]) -> Base58CryptoHash {
 pub extern "C" fn store() {
     env::setup_panic_hook();
     let contract: SputnikDAOFactory = env::state_read().expect("Contract is not initialized");
+    contract.assert_owner();
```

Comment:
This check will prevent random users to store blobs in the storage. Only the factory owner should be able to store new versions of the contract :)

Reply:
I agree:

https://github.com/near-daos/sputnik-dao-contract/pull/69#pullrequestreview-849850581

🚀  Thank you